indigo_app/templates/indigo_api/commencement_form.html

```diff
@@ -0,0 +1,145 @@
+{% load i18n %}
+
+<form
+    hx-post="{% url 'work_commencement_edit' frbr_uri=work.frbr_uri pk=commencement.id %}"
+    hx-target="body"
```

Comment:
As suggested, you can safely re-render the whole page, so these would change to `method="post" action="(url)"`

Reply:
and the success handler for the UpdateView would redirect to the commencements detail page for the work.